scripts/bootstrap.sh

```diff
@@ -99,9 +99,9 @@ sudo wget http://download.osmand.net/latest-night-build/OsmAndMapCreator-main.zi
 sudo unzip /var/lib/eventkit/OsmAndMapCreator/OsmAndMapCreator-main.zip
 sudo rm -f OsmAndMapCreator-main.zip
 
-sudo wget http://www.mkgmap.org.uk/download/mkgmap-r3691.zip
-sudo unzip mkgmap-r3691.zip
-sudo mv mkgmap-r3691 /var/lib/eventkit/
+sudo wget http://www.mkgmap.org.uk/download/mkgmap-r3693.zip
+sudo unzip mkgmap-r3693.zip
```

Comment:
I moved those dependencies to the top of the script. The bootstrap.sh and the Dockerfile_eventkit need to be updated separately, but it is just in those two spots. Additionally I just echoed the config file that needs to match, that way it gets updated as well.